### PR TITLE
Upgrade to polars 0.43

### PR DESCRIPTION
**Pull request by ayax79**
Upgrades the polars plugin to polars version 0.43

### CODE REVIEW

**Comment by sholderbach** [comment]
Thanks for going through those chores!

Should our `sqlparser` and `brotli` dependencies also be updated with this PR?

**Comment by ayax79** [comment]
> Thanks for going through those chores!
> 
> Should our `sqlparser` and `brotli` dependencies also be updated with this PR?

I updated sqlparser, I would prefer to not put brotli in this PR as it isn't part of the polars plugin. I can submit another PR for it.

File: Cargo.lock [hunk 1]
```diff
@@ -4666,17 +4704,18 @@ dependencies = [
  "polars-ops",
  "polars-plan",
  "polars-time",
+ "polars-utils",
  "rand",
  "serde",
  "serde_json",
- "sqlparser",
+ "sqlparser 0.49.0",
```
Comment (sholderbach) [inline Cargo.lock:4711]: I was thinking it would make sense to keep our `sqlparser` dependency on the same version as `polars-sql`?

**Comment by sholderbach** [comment]
Thanks! Let's go

**Pull request merged**